pkgs/by-name/ke/keycastr/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchurl,
+  unzip,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "keycastr";
+  version = "0.9.18";
+
+  src = fetchurl {
+    url = "https://github.com/keycastr/keycastr/releases/download/v${finalAttrs.version}/KeyCastr.app.zip";
+    hash = "sha256-q12c/W0yGoVL+wx+T/gaevx2P0Xwcv0a0FMv7bKfUnE=";
+  };
+
+  sourceRoot = ".";
+
+  nativeBuildInputs = [ unzip ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/{bin,Applications}
+    cp -r KeyCastr.app $out/Applications/
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://github.com/keycastr/keycastr";
+    description = "KeyCastr, an open-source keystroke visualizer";
+    license = lib.licenses.bsd3;
+    platforms = lib.platforms.darwin;
```

Comment:
Is this a unified app (or what do you darwin people call that)?

Reply:
Darwin people got me 🤣 yes, it is an universal app, for both Intel and Apple Silicon